applications/DEMApplication/python_scripts/KratosDEMAnalysis.py

```diff
@@ -0,0 +1,36 @@
+import KratosMultiphysics
+from KratosMultiphysics import Logger
+Logger.GetDefaultOutput().SetSeverity(Logger.Severity.INFO)
```

Comment:
I removed it

Reply:
It could be added to the launcher. IMO these lines should be written somewhere, even if we are calling the default severity. So anyone can change the severity without the need of looking at the wiki.